static/src/js/reducers/__tests__/dataQualitySummaries.test.js

```diff
@@ -9,7 +9,7 @@ describe('INITIAL_STATE', () => {
     expect(dataQualitySummariesReducer(undefined, action)).toEqual(initialState)
   })
 })
-
+// TODO: Are we using the provider-id and updated at values
```

Comment:
static/src/js/components/DataQualitySummary/DataQualitySummary.js, line 29

It looks like we only use the `id` and `summary`

Reply:
Removed these and also removed the name field which we were also not using